ethereum/contracts/bridge/Bridge.sol

```diff
@@ -217,6 +221,8 @@ contract Bridge is BridgeGovernance, ReentrancyGuard {
      *  @notice Initiate a transfer
      */
     function _transferTokens(address token, uint256 amount, uint256 arbiterFee) internal returns (BridgeStructs.TransferResult memory transferResult) {
+        require(evmChainId() == block.chainid, "invalid evmChainId");
```

Comment:
Is it necessary to have the check (or modifier) on outgoing transfers as well or could incoming be enough? (as forks will not be watched by guardians)

Reply:
I don't think it's necessary for outgoing transfers, because, as you mentioned, the fork won't even be watched.

Reply:
I think this is actualy a nice "protection" against users doing transfers that will go to nowhere and then complaining. Only concern is the SLOAD gas cost that is added to all outbound transfers (2100 gas).